exercises/all-your-base/all-your-base.spec.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+const Converter = require('./all-your-base');
+
+const converter = new Converter();
+
+describe('Converter', function () {
+
+  it('wrong input_base base less than two', function () {
+    expect(function () {
+      converter.convert([0]);
```

Comment:
To reflect better what line 9 describes, shouldn't this line be:

    converter.convert([0], 1);

Reading the test, it looks like `[0]` is the `inputBase` parameter, but in fact, it's `array` parameter.

What do you think? Do you think users will understand better this test adding the `inputBase` parameter as `1` (or another value less than `2`)?